### PR TITLE
main/rust: remap debug paths to rust-src

### PR DESCRIPTION
**Pull request by z-erica**

### CODE REVIEW

File: main/rust/template.py [hunk 1]
```diff
@@ -76,6 +76,13 @@ def post_patch(self):
     cargo.clear_vendor_checksums(self, "libc-0.2.155")
 
 
+def init_configure(self):
```
Comment (q66) [inline main/rust/template.py:79]: doesn't need to be a hook

Comment (nekopsykose) [reply main/rust/template.py:79]: mfs be like 'doesnt need to be x' without saying 'do y'

Comment (nekopsykose) [reply main/rust/template.py:79]: tool_flags=

Comment (q66) [reply main/rust/template.py:79]: just put it in toplevel tool_flags as i said on irc

File: main/rust/template.py [hunk 2]
```diff
@@ -31,6 +31,12 @@
 url = "https://rust-lang.org"
 source = f"https://static.rust-lang.org/dist/rustc-{pkgver}-src.tar.xz"
 sha256 = "0b9ca1e2e45b8a5f0b58db140af0dc92f8311faeb0ad883c5b71a72c02dc6e80"
+self.tool_flags = {
```
Comment (q66) [inline main/rust/template.py:34]: s/self.//

**Comment by nekopsykose** [comment]
because is updated but not in repo

**Comment by q66** [comment]
because there's 2 billion things queued (incl curl) while the builders are building chromium

**Comment by nekopsykose** [comment]
mfs be like 2 billion when the number is twelve

**Comment by nekopsykose** [comment]
yo can i get 2 billion units of food on my table rn i'm hungry

**Comment by q66** [comment]
nevermind, the libreoffice build is already done :^)

**Comment by nekopsykose** [comment]
where food

**Pull request merged**